ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -309,7 +310,9 @@ a database ( directory )
     // Populate the driver context with the scratch dir info from the repl context, so that the
     // temp dirs will be cleaned up later
     context.getFsScratchDirs().putAll(loadContext.pathInfo.getFsScratchDirs());
-    createReplLoadCompleteAckTask();
+    if (!HiveConf.getBoolVar(conf, REPL_DUMP_SKIP_IMMUTABLE_DATA_COPY)) {
```

Comment:
Please add this acknolegment check in the REPL_DUMP_SKIP_IMMUTABLE_DATA_COPY testcase also

Reply:
Added in the test case in recent commit.